downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md

```diff
@@ -47,105 +42,101 @@ A few backward-incompatible changes have been introduced during the Swan Lake Be
 
 Annotations are now supported on tuple members. Tuple members can be annotated using annotations declared with the `field` attachment point.
 
-```ballerina
-annotation annot on field;
+  ```ballerina
+  annotation annot on field;
 
-type T [int, @annot string];
-```
+  type T [int, @annot
+      string ];
+  ```
 
-Annotations are not allowed on the tuple rest descriptor. 
-
-### Improvements
+  Annotations are not allowed on the tuple rest descriptor. 
 
 ### Bug fixes
 
-- Annotation values of fields of record type descriptors that are not defined with a type definition are now accessible at runtime
+- Annotation values of fields of record type descriptors that are not defined with a type definition are now accessible at runtime.
```

Comment:
Why only this bug is special cased?

Reply:
It is in a way a behavioural change (we get annotation values we previously didn't get at runtime), thus we've emphasized it over other bug fixes.

We used to this for bug fixes that are breaking changes.

Reply:
Is that a breaking change? Then shouldn't we add a topic as breaking changes? @MaryamZi